### ISSUE TITLE
Streamline our Documentation structure in Help and How-to

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#18049]
@vshcherb @DmitryAlexei 

I believe the structure of our documentation at osmand.net/docs is becoming too complex to both properly maintain, and make it easy for users to positively find the correct explanation of a specific feature.

There are multiple sections with detailed explanations on the very same topic, setting, item or feature, and often contradictory due to lack of coherent maintenance. (I do a lot of maintenance and am surprised daily by yet another outdated paragraph explaining something from scratch in yet another article.)

This mainly results from us providing _app-descriptive articles_ and _use-case specific narratives_ in parallel, and in all take the freedom to describe from scratch things already documented elsewhere. (There are at least 5 different descriptions of the "Animations" settings, none of them entirely correct. Many not listed in the docs search, because they have no own headline.)

A way out could be we divide the documentation into two main sections:

- In a **Help** section we strictly use a screen-by-screen, item-by-item approach in a sense that this contains _the authoritative (and sole)  description of each screen, dialog  and item_ (menu line item, widget, button, option).
  - Basically every point in the app should be linkable to exactly one section in this Help documentation, describing what an element does.
  - The Help portion of our documentation is strictly hierarchical, contains exactly one article for each top app screen. Which links to one article for each dialog or settings screen it provides, which again contain exactly one authoritative section describing each setting items, buttons, widgets, actions.
  - Gaps in the Help documentation (i.e. a missing section on a new menu item) are in this fashion comparatively easy to detect.

- In a second strain of our documentation we can work use-case based (e.g. **How to**). But there we do _never duplicate descriptions of any feature_! We only always only provide a link to its (sole) primary description.

If you like this concept, I could work on some suggestions of target structure, using our current articles as starting point to describe where each would fall.

**Comment by sonora** [comment]
@DmitryAlexei Ok, cool! I'll take some time to work on a concept:  A proposed structure, marking also where existing articles would fit in, and which articles may be  missing.

Will post it here, then we can take a look together. 😉

**Comment by Sonwon1** [comment]
> @DmitryAlexei Ok, cool! I'll take some time to work on a concept: A proposed structure, marking also where existing articles would fit in, and which articles may be missing.
> 
> Will post it here, then we can take a look together. 😉

A few possible sections to consider,

Quick Start Guide
How to guides

Technical guides for more complex actions like installing maps from sqlitedb files.

**Comment by sonora** [comment]
It's a huge task, and getting it right will take some time, so please bear with me. I have started an inventory of what we have, and am drafting a more intelligible structure to organize the topics, de-duplicate them, and arrange them.

What I can already say is, starting at the very top navigation level of your web site:

1. We have (in several places) a link called "Map", which actually in many cases links to our online, web-based map service https://osmand.net/map/. This is confusing and not helpful for all users looking for app support/help. I advise to rename, maybe to something like "**Online Map**" or "**OsmAnd Map on the Web**".
2. We have a top level category we call "Docs", too generic to be helpful. I advise to rename to "**App User Guide**". We could later diversify into **"App Reference"**, **"App User Guide"**, **"App Technical Documentation"**.
3. Not sure if we want **"Build it"** on our top level hierarchy? Could potentially be moved under "Technical Articles"?

**Comment by sonora** [comment]
Here's the current site map of the osmand.net documentation - more or less. With the black stuff being what we have, and the green stuff the rudimentary of how to re-arrange... [osmand-net-docs.xlsx](https://github.com/osmandapp/OsmAnd/files/13323029/osmand-net-docs.xlsx)

**Comment by Sonwon1** [comment]
> Here's the current site map of the osmand.net documentation - more or less. With the black stuff being what we have, and the green stuff the rudimentary of how to re-arrange... [osmand-net-docs.xlsx](https://github.com/osmandapp/OsmAnd/files/13323029/osmand-net-docs.xlsx)

This is very good!

One thing I noticed, I did not see an import / export route section, if I missed it my apologies.

**Comment by vshcherb** [comment]
Took a while to reply but I believe ```App reference ``` should cover all screens and ```User guides``` should be separate section with links to to ```App reference```

**Comment by sonora** [comment]
@vshcherb  Yes, Victor, that's precisely what I had in mind when I opened this Issue. 

Unfortunately, over time I have become hung up in other projects and can spend much less time on this topic than I had hoped to.

But since we seem to agree that that's the way to go to make our documentation more crisp, accessible, and de-,duplicated, I guess we can all over time start to work in this direction. In particular also those of our team who professionally regularly tend to our documentation tasks. Creating the corresponding top level structure of having an `App Reference` and a `User Guide` section would be a great next step.

**Comment by vshcherb** [comment]
Yes, I'm preparing next year to change approach how we create documentation to speedup ```maintenance``` phase, we're almost there cause we have all articles in place but they are not read through yet. So it's a bit fast to start writing Guides but definitely in plan for 2024. The most difficult part to do it in a regular manner.

**Comment by Sonwon1** [comment]
> Yes, I'm preparing next year to change approach how we create documentation to speedup `maintenance` phase, we're almost there cause we have all articles in place but they are not read through yet. So it's a bit fast to start writing Guides but definitely in plan for 2024. The most difficult part to do it in a regular manner.

If you need a proofreader I could help out.

**Comment by johnnew2019** [comment]
I'm a retired technical writer (25+ years of experience) and could help in some way.
